examples/benchmarks/bubblebench/common/src/types.ts

```diff
@@ -27,10 +27,11 @@ export interface IBubble {
 /**
  * @internal
  */
-export interface IClient {
+export interface IClient<out T extends readonly IBubble[] = readonly IBubble[]> {
 	clientId: string;
 	color: string;
-	bubbles: IBubble[];
+	// Make the bubbles read-only, as SharedTree ArrayNodes are not compatible with JavaScript arrays for writing purposes.
```

Comment:
This comment is no longer in the right place since its now about the T type parameter.